SECURITY.md

```diff
@@ -6,6 +6,7 @@ Patches for **Rokwire Building Blocks** in this repository will only be applied
 
 | Version  | Supported          |
```

Comment:
Need to change version in events.yaml file as well.

Reply:
fixed.